### DISCUSSION TITLE
Unable to Display Releases on Repository Start Page Despite Enabling Them - Help Needed!

### DISCUSSION THREAD

**Discussion by elderguardian** [community/community#62208] [Repositories]
### Select Topic Area

Question

### Body

# Problem
Hello community,

I have an issue with one of my [repositories](https://github.com/elderguardian/img2minecraft) where the releases are not showing up on the repository start page, even after enabling them and saving the settings. Here's what I've done so far:

1. Navigated to the repository.
2. Clicked on settings symbol in the right sidebar.
3. Enabled the "Releases" checkbox under the "Include in the home page" section.
4. Clicked "Save changes" to confirm the settings.

After clicking the submit button, the page tells me that my settings have been saved. But, the releases are not visible on the main repository page. I can only find them if I go to the "Tags" section, which is not what I expected or wanted.

**Thanks you in advance for your time and assistance.**

## Sidebar
<img width="200" alt="example" src="https://github.com/community/community/assets/129489839/964fc643-26b5-4233-bea5-c8570d9688c1">

**Comment by MKJM2** [comment]
Hi! I've run into the same issue with one of my repositories. Did you manage to get it working?

**Reply by elderguardian** [reply]
Hello, all three sections (the Releases, Packages, and Environments) are now visible without my configuration. I do not know what exactly enabled them, but I created the first tag and enabled GitHub Pages for the repository yesterday. I am assuming they were enabled automatically after GitHub found content for them.

**Reply by tajiknomi** [reply]
Got to your Repository > settings (wheel icon) > Releases (tick)

![Capture](https://github.com/user-attachments/assets/92c60dd7-2297-44bc-b061-dd8dbdae4ec7)

**Comment by bergel** [comment]
I just bumped into this issue. You need to create a tag to create a release.
